protostar/cairo/cairo_function_runner_facade.py

```diff
@@ -42,6 +49,61 @@ def run_from_offset(
                 **kwargs,
             )
 
+    # pylint: disable=unused-argument
+    @staticmethod
+    def run_as_main(
+        function_runner: CairoFunctionRunner,
+        entrypoint: Union[str, int],
+        *args,  # type: ignore
+        typed_args: Optional[bool] = False,
+        hint_locals: Optional[dict[str, Any]] = None,
+        static_locals: Optional[dict[str, Any]] = None,
+        run_resources: Optional[RunResources] = None,
+        verify_secure: Optional[bool] = None,
+        apply_modulo_to_args: Optional[bool] = None,
+    ):
+        """
+        Runs the program from the given entrypoint.
+
+        Additional params:
+        typed_args - If true, the arguments are given as Cairo typed NamedTuple generated
+          with CairoStructFactory.
+        verify_secure - Run verify_secure_runner to do extra verifications.
+        apply_modulo_to_args - Apply modulo operation on integer arguments.
+        """
+        if hint_locals is None:
+            hint_locals = {}
+
+        if verify_secure is None:
+            verify_secure = True
+
+        if apply_modulo_to_args is None:
+            apply_modulo_to_args = True
```

Comment:
This logic can be moved to default arguments' values.

Reply:
Idk why it is like this in run_from_entrypoint 🤔

Reply:
I will leave it as it is for consistency though

Reply:
Ok, sure